### DISCUSSION TITLE
开代理好像有些打不开

### DISCUSSION THREAD

**Discussion by Jinlingsang** [xiaohucode/yidaRule#242] [Q&A]
那些需要挂梯子的我用vpn时可以打开，但是用圈X就不行，开全局代理，直连都没用。![image](https://github.com/user-attachments/assets/2d9006b8-4081-4942-b79d-0df5a1002138)

**Comment by Jinlingsang** [comment]
之前能打开，现在又打不开了![IMG_6658](https://github.com/user-attachments/assets/bd9cebbf-7522-4637-b1f0-3c152fd9692f)

**Reply by bbuus** [reply]
现在有解决办法吗

**Comment by Jinlingsang** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
